nifi-nar-bundles/nifi-box-bundle/nifi-box-processors/src/main/resources/docs/org.apache.nifi.processors.box.FetchBoxFiles/additionalDetails.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<head>
+    <meta charset="utf-8" />
+    <title>ListBoxFiles</title>
+
+    <link rel="stylesheet" href="../../../../../css/component-usage.css" type="text/css" />
+</head>
+<body>
+<p>This processor requires a pre-configured App under the User owning the files that are being fetched.</p>
+<p>The App should have the following configuration:</p>
+<ul>
+    <li>If you create a new App, select 'Server Authentication (with JWT)' authentication method.
+        If you want to use an existing App, chose one with 'OAuth 2.0 with JSON Web Tokens (Server Authentication)' as Authentication method.</li>
+    <li>Should have a Client ID and Client Secret</li>
+    <li>App Access Level should be 'App + Enterprise Access'</li>
+    <li>Application Scopes should have 'Read all files' enabled</li>
+    <li>Advanced Features should have 'Generate user access tokens' enabled</li>
```

Comment:
Besides 'Generate user access tokens', the 'Make API calls using the as-user header" needs to be enabled, otherwise the API returns 403 error.